### PR TITLE
Perf fix2

### PR DESCRIPTION
**Pull request by oleks-rip**

### CODE REVIEW

File: src/xbwd/client/WebsocketClient.cpp [hunk 1]
```diff
@@ -237,4 +235,57 @@ WebsocketClient::onReadDone()
...
+            processingQueue_.swap(receivingQueue_);
+        }
+
+        auto const x = processingQueue_.size();
+        if (x > 10000)
+        {
+            JLOGV(
+                j_.debug(),
+                "WebsocketClient::runCallbacks",
+                jv("size of queue", processingQueue_.size()));
```
Comment (pwang200) [inline src/xbwd/client/WebsocketClient.cpp:258]: Use `x`, instead of `processingQueue_.size()`

**Review by pwang200 — APPROVED** [review]
:+1:

File: src/xbwd/client/WebsocketClient.cpp (hunk 1 above)
Comment (pwang200) [reply src/xbwd/client/WebsocketClient.cpp:258]: `x` probably should have a better name.

**Review by seelabs — COMMENTED** [review]
:+1:

**Pull request merged**